_includes/cwl/09-array-inputs/array-inputs.cwl

```diff
@@ -27,5 +27,8 @@ inputs:
       separate: false
       position: 4
 
-outputs: []
+outputs:
+  output:
```

Comment:
Please use a less generic name than `output`; people new to CWL will get confused as it looks similar to the CWL keyword `outputs`

Reply:
Thank you for your comment.
I will replace it with `example_out`.